pkg/migrator/testdata/schema.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE key_packages(
+	sequence_id BIGSERIAL PRIMARY KEY,
+	installation_id BYTEA NOT NULL,
+	key_package BYTEA NOT NULL
+);
+
+CREATE TABLE group_messages(
+	id BIGSERIAL PRIMARY KEY,
+	created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+	group_id BYTEA NOT NULL,
+	data BYTEA NOT NULL,
+	group_id_data_hash BYTEA NOT NULL
```

Comment:
Martin now has the `is_commit` field in this DB, and I have a [PR open](https://github.com/xmtp/xmtp-node-go/pull/442) to add a new `should_push` and `sender_hmac` coming very soon.

Sorry to make you have to hit a moving target. Should be the end of schema updates soon.